db/post_migrate/20220321235638_drop_chat_message_post_connections_table.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'migration/table_dropper'
+
+class DropChatMessagePostConnectionsTable < ActiveRecord::Migration[6.1]
+  DROPPED_TABLES ||= %i{
+    chat_message_post_connections
+  }
+
+  def up
+    DROPPED_TABLES.each do |table|
```

Comment:
Why the loop, are you expecting DROPPED_TABLES to be populated elsewhere?

Reply:
No idea TBH, this just seems to be the pattern we have adopted for them, even for single tables:

https://github.com/discourse/discourse/blob/31bc756037834b531750167f029c734e12040db9/db/migrate/20201110110952_drop_github_user_infos.rb

Doesn't really make sense though you're right, will get rid of the loop and the const, it's not necessary

Reply:
This is not well documented by we actually need the `DROPPED_TABLES` constants for backup/restore. See https://github.com/discourse/discourse/commit/7cb51d0e4069ef982919e9f005e6bb18da04ebed for more details. In core, we actually have a spec to catch this.